pkg/command/plugin_test.go

```diff
@@ -286,7 +286,7 @@ func TestInstallPlugin(t *testing.T) {
 			centralRepoFeature: "true",
 			args:               []string{"plugin", "install", "all"},
 			expectedFailure:    true,
-			expectedErrorMsg:   "the 'all' argument is not longer supported. You must install each plugin individually",
+			expectedErrorMsg:   "the 'all' argument can only be used with the --group or --local flags",
```

Comment:
not blocking, maybe something for following discussion as well.

feels like supporting 'all', but only as a redundant argument is worth considering.

I am guessing when a group is used, installing all the plugins defined for the group is going to be the overwhelming usage pattern, so do we want to consider also supporting not defaulting to all when group is provided and no plugin argument is passed?

Reply:
Yeah, it was bothering me too. I don't know why I didn't think about defaulting to `all`. Thanks for the suggestion. I will do it.